org.scala-ide.sdt.core/src/org/scalaide/core/internal/project/ClasspathManagement.scala

```diff
@@ -45,6 +45,7 @@ import scala.util.Success
 import org.scalaide.core.IScalaClasspath
 import org.scalaide.core.ScalaInstallationChange
 import org.eclipse.jface.preference.IPersistentPreferenceStore
+import scala.tools.nsc.Settings
```

Comment:
I know it's not in scope of this PR, but those imports are terrible, could you please organize them?